src/main/java/org/terracotta/statistics/derived/histogram/BarSplittingBiasedHistogram.java

```diff
@@ -513,7 +513,7 @@ public double count() {
 
     @Override
     public String toString() {
-      return "[" + minimum() + " --" + count() + " [height=(" + count() / (maximum() - minimum()) + ")-> " + maximum() + "]";
+      return "[" + minimum() + " (" + count() + " / " + count() / (maximum() - minimum()) + ") " + maximum() + "]";
```

Comment:
what would you suggest to improve ? This one otherwise ?

`return "[" + minimum() + "  -- " + count() + " -- height=(" + count() / (maximum() - minimum()) + ") -> " + maximum() + "]";`

Reply:
In the following form `[0.0 (10.0 / 5.0) 2.0]` the `/` is confusing since it implies division.  Now I know that this does make sense technically... but it's not something we want people thinking about.  I'd either be explicit about what the count and 'height/density/frequency' are, or express the real math explicity: `[0.0 (2.0 * 5.0 = 10.0) 2.0]`.  Does that make sense?